monitoring/ceph-mixin/dashboards_out/ceph-cluster-advanced.json

```diff
@@ -3788,5 +3808,6 @@
    },
    "timezone": "",
    "title": "Ceph Cluster - Advanced",
+   "uid": "dn13KBeTv",
```

Comment:
Do we need this static uid? I did test on my existing system and all dashboard upgraded but not ceph-cluster-advance dashboard.

I am assuming this might be the thing that is message it up. Not finding any other thing.

Reply:
All other boards already had random static IDs so I added one for this board as well to align them.
This is what Grafana has to say about UIDs: https://grafana.com/docs/grafana/latest/administration/provisioning/#reusable-dashboard-urls

I don't mind changing things around, my goal was simply to align things as much as possible.

Reply:
Ack, we can have it, Maybe we didn't have it earlier is causing this issue. Lets continue having it.